sql/src/main/java/io/crate/metadata/doc/DocIndexMetaData.java

```diff
@@ -136,6 +139,7 @@
         }
         versionCreated = getVersionCreated(mappingMap);
         versionUpgraded = getVersionUpgraded(mappingMap);
+        closed = getClosed(mappingMap);
```

Comment:
`IndexMetaData` has a State enum that has close/open, is that something different?